distribution/docs/src/main/resources/_contents/_running/monitoring-contents.adoc

```diff
@@ -36,6 +36,11 @@ A useful way to debug and detect issues is to change the log level:
 
 `log4j.rootLogger=DEBUG, out, osgi:VmLogAppender`
 
+[NOTE]
```

Comment:
After discussing this suggestion with you, we decided that no other documentation is needed right now.